src/test/java/com/conveyal/gtfs/TestUtils.java

```diff
@@ -11,11 +11,14 @@
 import java.io.IOException;
 import java.sql.Connection;
 import java.sql.DriverManager;
+import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipOutputStream;
 
 import static com.conveyal.gtfs.util.Util.randomIdString;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
Any specific reason to use this test library over junit?

Reply:
JUnit4 assertThat (https://junit.org/junit4/javadoc/latest/org/junit/Assert.html#assertThat(java.lang.String,%20T,%20org.hamcrest.Matcher) is deprecated. This is the suggested replacement.